### DISCUSSION TITLE
Copy IPAM from one netbox to another

### DISCUSSION THREAD

**Discussion by joubbi** [netbox-community/netbox#8554] [Q&A]
I have a netbox installation used for production.
It has a bunch of prefixes, VRFs, VLANs and such already.

I have another netbox installation with prefixes, VLANs and VRFs.

I would like to copy everything from the other netbox to the production netbox, without losing the existing information.
In other words, I need to add the IPAM from one instance of netbox to an existing netbox.
What is the easiest way? Is there a tool/script for this somewhere that I have not found yet?

**Comment by michaelklose** [comment]
Have you tried csv export and import?
